packages/person/src/types.ts

```diff
@@ -11,44 +10,65 @@ export type Position = {
     name: string;
   };
 };
-
-export type Manager = {
-  azureUniqueId: string;
-  name?: string;
-  pictureSrc?: string;
-  department?: string;
-  accountType?: PersonAccountType;
-};
-
 export type PersonDetails = {
-  azureId: string;
```

Comment:
using this type in resolver

Reply:
No so sure about this, the api result all has azureUniqueId, so maybe better to update resolver also use that?
We can take a look when we get closer.

Reply:
That's maybe a problem, since person api has azureId, and search has azureUniqueId

Reply:
we can use both